### PR TITLE
fix: keychain lib still broken

### PR DESCRIPTION
**Pull request by kyranjamie**

### CODE REVIEW

File: packages/keychain/package.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "@blockstack/keychain",
-  "version": "0.12.2",
+  "version": "0.13.0-beta",
```
Comment (hstove) [inline packages/keychain/package.json]: Can you revert this change? Otherwise it'll mess up the auto-publish setup. And, for local development, try out yalc. It is pretty nice once you figure out the commands.

**Review by hstove — CHANGES_REQUESTED** [review]

File: packages/keychain/package.json (hunk 1 above)
Comment (kyranjamie) [reply packages/keychain/package.json]: Just tried it.

Does yalc add commits? Not sure where the version named commit came from.
